src/main/java/crazypants/enderio/machine/MachineObject.java

```diff
@@ -33,10 +48,47 @@
 //  blockCapBank(BlockCapBank.class),
   blockCrafter(BlockCrafter.class),
   blockEnchanter(BlockEnchanter.class),
-  
+
+  //Obelisks
+  blockAttractorObelisk(BlockAttractor.class),
+  blockAversionObelisk(BlockAversionObelisk.class),
+  blockInhibitorObelisk(BlockInhibitorObelisk.class),
+  blockRelocatorObelisk(BlockRelocatorObelisk.class),
+  blockWeatherObelisk(BlockWeatherObelisk.class),
+  blockExperienceObelisk(BlockExperienceObelisk.class),
+
+  //Painter
+  blockPainter(BlockPainter.class),
+  blockPaintedFence(BlockPaintedFence.class),
+  blockPaintedStoneFence(BlockPaintedFence.class, "create_stone"),
+  blockPaintedFenceGate(BlockPaintedFenceGate.class),
+  blockPaintedWall(BlockPaintedWall.class),
+  blockPaintedStair(BlockPaintedStairs.class),
+  blockPaintedStoneStair(BlockPaintedStairs.class, "create_stone"),
+  blockPaintedSlab(BlockPaintedSlab.class),
+  blockPaintedDoubleSlab(BlockPaintedSlab.BlockPaintedDoubleSlab.class), //TODO Fixed with proper Classes
+  blockPaintedStoneSlab(BlockPaintedSlab.BlockPaintedDoubleSlab.class),
+  blockPaintedStoneDoubleSlab(BlockPaintedSlab.BlockPaintedDoubleSlab.class),
+  blockPaintedGlowstone(BlockPaintedGlowstone.class),
+  blockPaintedGlowstoneSolid(BlockPaintedGlowstone.class, "create_solid"),
+  blockPaintedCarpet(BlockPaintedCarpet.class),
+  blockPaintedPressurePlate(BlockPaintedPressurePlate.class),
+  blockPaintedRedstone(BlockPaintedRedstone.class),
+  blockPaintedRedstoneSolid(BlockPaintedRedstone.class),
+  blockPaintedStone(BlockPaintedStone.class),
+  blockPaintedWoodenTrapdoor(BlockPaintedTrapDoor.class, "create_wooden"),
+  blockPaintedIronTrapdoor(BlockPaintedTrapDoor.class, "create_iron"),
+  blockPaintedDarkSteelTrapdoor(BlockPaintedTrapDoor.class, "create_dark"),
+  blockReservoir(BlockReservoir.class),
+
   blockFarmStation(BlockFarmStation.class),
-  
-  ;
```

Comment:
this was on purpose so that adding an entry to the end only adds a line instead of adding and changing (not important, just FYI)